Scripts/script-DBotPredictPhishingLabel.yml

```diff
@@ -0,0 +1,71 @@
+commonfields:
+  id: DBotPredictPhishingLabel
+  version: -1
+name: DBotPredictPhishingLabel
+script: |-
+  import json
+  args = demisto.args()
+  args['inputText'] = args.get('emailSubject','') + ' ' + args.get('emailBody','')
+  res = demisto.executeCommand('DBotPredictTextLabel', demisto.args())
+  res = res[-1]
+  old_context_key_prefix = 'DBotPredictTextLabel'
+  new_context_key_prefix = 'DBotPredictPhishingLabel'
+  ec = res.get('EntryContext', {}) or {}
+  for key in ec:
+      if key.startswith(old_context_key_prefix):
+          new_key = new_context_key_prefix + key.lstrip(old_context_key_prefix)
+          ec[new_context_key_prefix] = ec.pop(old_context_key_prefix)
+  res['EntryContext'] = ec
+  demisto.results(res)
+type: python
+tags:
+- ml
+- phishing
+comment: |-
+  Predict text label using pre-trained machine learning phishing model.
+  Notice you have to run training playbook successfully first.
+enabled: true
+args:
+- name: warningThreshold
+  description: If the number of words in the input  text smaller then threshold a
+    warning will be displayed.
+  defaultValue: "30"
+- name: modelListName
```

Comment:
where is the default value for this?

Reply:
I prefer leave this empty, and there the customer must explicit set. It cuz a bug in orange which was hard to detect